packages/conf-perl-string-shellquote/conf-perl-string-shellquote.2/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "2.0"
+maintainer: "chetsky@gmail.com"
+homepage: "https://www.perl.org/"
+bug-reports: "chesky@gmail.com"
+license: "GPL-1.0-or-later"
+authors: "Larry Wall et. al."
+depends: [
+  "conf-perl"
+]
+build: [
+  ["perl" "--version"]
+  ["perl" "-MString::ShellQuote" "-e" "1"]
+]
+depexts: [
+  ["libstring-shellquote-perl"] {os-family = "debian"}
+  ["perl-string-shellquote"] {os-distribution = "alpine"}
+  ["perl-String-ShellQuote"] {os-distribution = "centos"}
+  ["perl-String-ShellQuote"] {os-distribution = "ol"}
+  ["perl-String-ShellQuote"] {os-family = "suse"}
+  ["perl-String-ShellQuote"] {os-family = "fedora"}
+  ["perl-string-shellquote"] {os-family = "arch"}
+  ["p5-string-shellquote"] {os-family = "macports"}
```

Comment:
```suggestion
  ["p5-string-shellquote"] {os-distribution = "macports" & os = "macos"}
```

Reply:
Applying your suggestion.

As long as you're looking, I have a workaround (from Blaisorblade) for the macos+homebrew problem: just disable testing on that platform.  It involves only modifying the opam file -- should I roll a fresh release, or just modify the opam file in opam-repository ?

Reply:
If it is just about amending the test line on the opam file, that is fine to do here without a new release (as long as the tarball remains the same): you could do `{with-test & os != "macos"}`.